src/utils/mcp-config.ts

```diff
@@ -0,0 +1,222 @@
+    clack.log.success(
+      chalk.cyan(path.join('.cursor', 'mcp.json')) + ' created.',
+    );
+    return;
+  }
+  try {
+    const updated = { ...existing } as CursorMcpConfig;
+    updated.mcpServers = updated.mcpServers || {};
+    updated.mcpServers['Sentry'] = {
+      url: SENTRY_MCP_URL,
+    };
+    await writeJson(file, updated);
+    clack.log.success('Updated .cursor/mcp.json');
+  } catch {
+    throw new Error('Failed to update .cursor/mcp.json');
+  }
+}
+
+async function addVsCodeMcpConfig(): Promise<void> {
+  const file = path.join(process.cwd(), '.vscode', 'mcp.json');
+  const existing = await readJsonIfExists(file);
+  if (!existing) {
+    await writeJson(file, JSON.parse(getVsCodeMcpJsonSnippet()));
+    clack.log.success(
+      chalk.cyan(path.join('.vscode', 'mcp.json')) + ' created.',
+    );
+    return;
+  }
+  try {
+    const updated = { ...existing } as VsCodeMcpConfig;
+    updated.servers = updated.servers || {};
+    updated.servers['Sentry'] = {
+      url: SENTRY_MCP_URL,
+      type: 'http',
+    };
+    await writeJson(file, updated);
+    clack.log.success('Updated .vscode/mcp.json');
+  } catch {
+    throw new Error('Failed to update .vscode/mcp.json');
+  }
+}
+
+async function addClaudeCodeMcpConfig(): Promise<void> {
+  const file = path.join(process.cwd(), '.mcp.json');
+  const existing = await readJsonIfExists(file);
+  if (!existing) {
+    await writeJson(file, JSON.parse(getClaudeCodeMcpJsonSnippet()));
+    clack.log.success(chalk.cyan('.mcp.json') + ' created.');
+    return;
+  }
+  try {
+    const updated = { ...existing } as ClaudeCodeMcpConfig;
+    updated.mcpServers = updated.mcpServers || {};
+    updated.mcpServers['Sentry'] = {
+      url: SENTRY_MCP_URL,
+    };
+    await writeJson(file, updated);
+    clack.log.success('Updated .mcp.json');
+  } catch {
+    throw new Error('Failed to update .mcp.json');
+  }
+}
+
+/**
+ * Offers to add a project-scoped MCP server configuration for the Sentry MCP.
+ * Supports Cursor, VS Code, and Claude Code.
+ */
+export async function offerProjectScopedMcpConfig(): Promise<void> {
+  const shouldAdd = await abortIfCancelled(
+    clack.select({
+      message:
+        'Optionally add a project-scoped MCP server configuration for the Sentry MCP?',
+      options: [
+        { label: 'Yes', value: true },
+        { label: 'No', value: false, hint: 'You can add it later anytime' },
+      ],
+      initialValue: true,
+    }),
+  );
+
+  if (!shouldAdd) {
+    return;
+  }
+
+  type EditorChoice = 'cursor' | 'vscode' | 'claudeCode';
+  const editor: EditorChoice = await abortIfCancelled(
+    clack.select({
+      message: 'Which editor do you want to configure?',
+      options: [
+        { value: 'cursor', label: 'Cursor (project .cursor/mcp.json)' },
+        { value: 'vscode', label: 'VS Code (project .vscode/mcp.json)' },
+        { value: 'claudeCode', label: 'Claude Code (project .mcp.json)' },
```

Comment:
(l) If we end up attempting editor auto-detection, we can pre-select the editor we auto-detected (and fall back to the first or whatever if we fail)